tests/svg.sil

```diff
@@ -0,0 +1,4 @@
+\begin{document}
```

Comment:
I don't think we need this test. We do of course need to test SVG, but if we're testing specific aspects of SVG support having a separate test for SVG itself is not only duplicate work, it adds to the maintenance burden and makes it more complicated than it needs to be to review and update test fixtures when they change. Lets drop everything except the very specific minimized tests.

Reply:
Since @skef looked at the C code and helped me fix it, probably worrying overly about memory management isn't such a big concern; he's really good at it and I trust his review. If it's OK with you I'll just scrap this test.

Reply:
Sure. We can always add a test when we have a better idea what exactly needs testing. With a `busted` test we could assert the specific things you mention (not crashing, not having nulls, being ASCII, being at least X long. I'm happy to work up the test for that if there was a clear need to, but it also seems like most or all of those will get tested as side effects already anyway.

Dropping it for now is fine.